jenkins/dockerized-e2e-runner.sh

```diff
@@ -30,7 +30,7 @@ mkdir -p "${HOST_ARTIFACTS_DIR}"
 : ${JENKINS_GCE_SSH_PRIVATE_KEY_FILE:='/var/lib/jenkins/gce_keys/google_compute_engine'}
 : ${JENKINS_GCE_SSH_PUBLIC_KEY_FILE:='/var/lib/jenkins/gce_keys/google_compute_engine.pub'}
 
-KUBEKINS_E2E_IMAGE_TAG='v20161004-f3120c3'
+KUBEKINS_E2E_IMAGE_TAG='v20161007-34648c8'
```

Comment:
Ugh well it was the git hash of repo with head of e2e-runner.sh + the version change... which apparently is no longer true given various rebases.

Personally given that this image version is already tested as working I'd prefer not to create it just to make the hash match (I think that will be slower or else more risky)